tests/lms/models/group_info_test.py

```diff
@@ -0,0 +1,82 @@
+import pytest
+from sqlalchemy.exc import IntegrityError
+
+from lms.models import ApplicationInstance, GroupInfo
+
+
+class TestGroupInfo:
+    def test_persist_and_retrieve(self, application_instance, db_session, group_info):
+        db_session.add(group_info)
+
+        retrieved_group_info = db_session.query(GroupInfo).one()
```

Comment:
In this case is this getting persisted to a test db, or just in RAM?

Reply:
It actually gets sent to Postgres: `db_session.add(...)` adds it to the sqlalchemy session (which doesn't necessarily send it to Postgres yet) but then the following `db_session.query(...)` will cause an sqlalchemy _autoflush_ to happen (all queries cause sqlalchemy to flush all SQL statements to Postgres). Although the SQL statements do get sent to Postgres (and Postgres would crash if there was anything wrong with them) we never do ask Postgres to _commit_ the transaction though